app/controllers/hubstats/deploys_controller.rb

```diff
@@ -5,21 +5,22 @@ class DeploysController < ApplicationController
 
     def index
       #deploy_id = Hubstats::Deploy
```

Comment:
Delete this commented out code unless you have a reason to keep it in the project.